### PR TITLE
[CNVS Upgrade] Fix broken unit tests

### PR DESCRIPTION
**Pull request by jfurrow**
Velocity won't properly report the status of the unit tests because this still relies on the `cnvs-upgrade/master` branch of both `reactjs-components` and the plugins repository, but you can see them passing locally:

### CODE REVIEW

**Comment by jfurrow** [comment]
ping @MatApple :)

File: src/js/components/Panel.js [hunk 1]
```diff
@@ -48,7 +50,8 @@ var Panel = React.createClass({
     let panelClasses = classNames(defaultClasses.panel, props.className);
 
     return (
-      <div className={panelClasses}>
+      <div className={panelClasses}
+        {...Util.omit(props, ['children', 'className', 'heading', 'footer'])}>
```
Comment (MatApple) [inline src/js/components/Panel.js]: Do we need to add extra "unknown" attributes?

Comment (jfurrow) [inline src/js/components/Panel.js]: I'm not sure what you mean, can you clarify?

Comment (jfurrow) [inline src/js/components/Panel.js]: We talked offline, but this is done!

**Pull request merged**